internal/models/sso.go

```diff
@@ -154,8 +154,9 @@ type SAMLRelayState struct {
 
 	RedirectTo string `db:"redirect_to"`
 
-	CreatedAt time.Time `db:"created_at" json:"-"`
-	UpdatedAt time.Time `db:"updated_at" json:"-"`
+	CreatedAt   time.Time  `db:"created_at" json:"-"`
+	UpdatedAt   time.Time  `db:"updated_at" json:"-"`
+	FlowStateID *uuid.UUID `db:"flow_state_id" json:"flow_state_id,omitempty"`
```

Comment:
If you put in a `FlowState *FlowState` here and the relevant `belongs_to` flags, it Pop should just give you the object, right?

Reply:
You're right-  edited to use  `belongs_to` accordingly.